architect.yml

```diff
@@ -33,8 +33,12 @@ services:
       '
     interfaces:
       main: 8000
+    liveness_probe:
+      command: curl --fail 0.0.0.0:8000
+      interval: 30s
+      failure_threshold: 3
     environment:
-      ALLOWED_HOST: .${{ ingresses.web.host }}
+      ALLOWED_HOST: '*'
```

Comment:
I'd prefer if you left this and updated the allowed hosts in settings.py:
```
ALLOWED_HOSTS = ['localhost', '0.0.0.0', os.environ.get('ALLOWED_HOST', '')]
```
Double check that it works.

Reply:
Looks good both locally and remotely